test_cfms/c_horiz_interp/test_horiz_interp_new.c

```diff
@@ -22,6 +22,37 @@
 Adapted from test_horiz_interp :: test_assign
 */
 
+void define_domain(int* domain_id);
+
+int test_conservative_new(int domain_id);
+
+int test_bilinear_new(int domain_id);
+
+// run both tests and also check their return statuses
+int main(){
+  int test_conserve, test_bilinear;
+
+  int domain_id = 0;
+
+  define_domain(&domain_id);
```

Comment:
This should have been changed a while ago before your PR.  cFMS_define_domains now returns a domain_id.  Could you change this function so that domain_id = define_domain()?

Reply:
I'm calling the `define_domain` function defined below instead of something from the actual library.

That function was using `cFMS_define_domains_easy` but was not setting the right domain identifier, should be fixed now but lmk if thats not what you had in mind.